Packs/CommonPlaybooks/Playbooks/playbook-CVE_Enrichment_-_Generic_v2.yml

```diff
@@ -229,24 +234,27 @@ tasks:
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
+    continueonerrortype: ""
+    isoversize: false
+    isautoswitchedtoquietmode: false
   "6":
     id: "6"
-    taskid: c08adb3a-b8c3-4a9b-8290-2f6dcda45d88
+    taskid: ac59954c-2ceb-46e7-8777-4836c7c4f2f8
     type: regular
     task:
-      id: c08adb3a-b8c3-4a9b-8290-2f6dcda45d88
+      id: ac59954c-2ceb-46e7-8777-4836c7c4f2f8
       version: -1
-      name: Get CVE info from "CVE Search"
+      name: Get CVE info from CIRCL CVE Search
       description: Returns CVE information by CVE ID.
       script: '|||cve'
       type: regular
       iscommand: true
-      brand: CVE Search
+      brand: ""
     nexttasks:
       '#none#':
       - '2'
     scriptarguments:
-      cve_id:
+      cve:
         complex:
           root: inputs.cve_id
     continueonerror: true
```

Comment:
why this is enabled? if we are using skip if unavailable?

Reply:
the error config is set to 'continue' if an error occurs since the API returns an API error even when the results are returned.